### PR TITLE
#2882 Add link to nightly documentation in README

### PR DESCRIPTION
**Pull request by barondim**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [x] the corresponding milestone was added in the ticket and in this PR
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ To get started with Dyna&omega;o you have different possibilities, depending on
 
 **All validated models are included into the [Dyna&omega;o Modelica library](https://github.com/dynawo/dynawo/releases/download/v1.4.1/Dynawo_Modelica_library_v1.4.1.zip). Don't hesitate to open it in [OpenModelica](https://www.openmodelica.org/) to see what are the available models and simulate the simple full Modelica illustrative examples we provide.**
 
-**In addition, we also provide an example directory containing validated test cases for DynaFlow, DynaWaltz and DynaSwing. Don't hesitate to simulate them and open the [associated documentation](https://github.com/dynawo/dynawo/releases/download/v1.4.1/DynawoDocumentation.zip).**
+**In addition, we also provide an example directory containing validated test cases for DynaFlow, DynaWaltz and DynaSwing. Don't hesitate to simulate them and open the [associated documentation](https://github.com/dynawo/dynawo/releases/download/nightly/DynawoDocumentation.pdf).**
```
Comment (rosiereflo) [inline README.md:93]: we want to keep the zip of the latest release here

File: README.md [hunk 2]
```diff
@@ -349,7 +349,7 @@ You can have more information on how to use Docker to build and try Dyna&omega;o
 
 <a name="documentation"></a>
 ## Dyna&omega;o Documentation
-You can download Dyna&omega;o documentation [here](https://github.com/dynawo/dynawo/releases/download/v1.4.1/DynawoDocumentation.zip).
+You can download Dyna&omega;o documentation [here](https://github.com/dynawo/dynawo/releases/download/nightly/DynawoDocumentation.pdf).
```
Comment (rosiereflo) [inline README.md:352]: I wuld propose 'nightly documentation'

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (gautierbureau) [reply README.md]: we don't want to keep the two links ?

Comment (rosiereflo) [reply README.md]: If I'm not mistaken we have one link at the beginning which is the release one, and this one in the middle if forthe nightly, maybe we could reorganize a bit

Comment (rosiereflo) [reply README.md]: nope you're right the is the only link available so let's provide both

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**